fea-rs/src/compile/tables/name.rs

```diff
@@ -7,10 +7,13 @@ use write_fonts::{read::tables::name::Encoding, types::NameId};
 
 use crate::compile::tags::{MAC_PLATFORM_ID, WIN_PLATFORM_ID};
 
+pub(crate) const FIRST_NON_RESERVED_NAME_ID: u16 = 256;
+
 #[derive(Clone, Debug)]
 pub(crate) struct NameBuilder {
     records: Vec<(NameId, NameSpec)>,
-    last_anon_id: u16,
+    // the last used non-reserved nameid value.
+    last_nonreserved_id: u16,
```

Comment:
Optional, making this be _next_ rather than last even when there was no last would make more sense to me?

Reply:
This is just a rename; changing the semantics causes breakages elsewhere (I had tried this)